test/core/binary.wast

```diff
@@ -183,220 +183,6 @@
   "integer representation too long"
```

Comment:
This test is duplicated at https://github.com/WebAssembly/spec/blob/f607ec9a776a9e3425639f3cbe9b2d96bdc16b60/test/core/binary.wast#L372

In this file I generally deleted the first occurrence and left the second occurrence (because it looked like more tests in the second part). For this test I did the same, but GitHub here shows the diff as if I removed the second occurrence (at line 373)